resources/js/src/app/components/itemList/filter/ItemFilterList.vue

```diff
@@ -1,3 +1,77 @@
+<template>
```

Comment:
This should not be needed anymore, after removing the container link. The FindologicFilterWrapper should be the replacement.

Reply:
So you mean all of its content should be in the wrapper component?

Reply:
Yes, right know both exist with some duplicated code.

FindologicFilterWrapper should be the component for all use cases. (Which is probably only one, after removing the container link)